src/Widgets/TerminalWidget.vala

```diff
@@ -489,6 +530,11 @@ namespace Terminal {
             return get_text_range (start_row, 0, output_end_row - 1, 1000, null, null) + "\n";
         }
 
+        private void get_cell_clicked (Gdk.EventButton event, out long row, out long col) {
+            row = (long)(event.y / get_char_height ());
```

Comment:
Just a minor suggestion: As the functions of the `Vte.Terminal` also use "absolute" row-values, would it maybe better to return "absolute" values in the `get_cell_clicked_function`?
```vala
row = (long)(event.y / get_char_height () + get_vadjustment ().get_value ());
```
Or is there a reason to prefer "relative" values?

Or will there

Reply:
OK, I have now used absolute values consistently.